admin-dev/themes/default/template/controllers/modules/modal_translation.tpl

```diff
@@ -30,9 +30,19 @@
 			<span class="caret"></span>
 		</button>
 		<ul class="dropdown-menu">
-			{foreach from=$module_languages item=language}
-			<li><a href="{$trad_link|escape:'html':'UTF-8'}{$language['iso_code']|escape:'html':'UTF-8'}#{$module_name|escape:'html':'UTF-8'}">{$language.name|escape:'html':'UTF-8'}</a></li>
-			{/foreach}
+			{if $new_translate_system }
```

Comment:
Hello @Azouz-Jribi 

If you introduce the `case 'AdminTranslationSf':` in link.php, what do you think about: 
(see with the other comment)

```twig
{foreach from=$module_languages item=language}
    <li>
        {if $new_translate_system }
            <a href="{$link->getAdminLink('AdminTranslationSf', true, [
              'lang' => $language.iso_code,
              'type' => 'modules',
              'selected' => $module_name,
              'locale' => $language.locale
            ])|escape:'html':'UTF-8'}">{$language.name|escape:'html':'UTF-8'}</a>
        {else}
            <a href="{$link->getAdminLink('AdminTranslations', true, [], [
              'type' => 'modules',
              'module' => $module_name,
              'lang' => $language.iso_code
            ])|escape:'html':'UTF-8'}">{$language.name|escape:'html':'UTF-8'}</a>
        {/if}
    </li>
{/foreach}
```